src/app/shared/components/template/components/button/button.component.html

```diff
@@ -1,11 +1,14 @@
 <ion-button
+  *ngIf="!variantIsCardPortrait"
```

Comment:
nit(non-blocking)
Would be good to add comments to separate out the different variant visualisations, particularly noting why the Portrait is so standalone/custom

Reply:
nit(blocking)
It would be better to use an ngswitch statement to define specific case behaviour and then default in case we chose to have more custom variants in the future (save lots of !isVariantA && !isVariantB etc.)